src/monitor/pgautofailover.sql

```diff
@@ -362,13 +378,16 @@ CREATE FUNCTION pgautofailover.current_state
    OUT group_id             int,
    OUT node_id              bigint,
    OUT current_group_state  pgautofailover.replication_state,
-   OUT assigned_group_state pgautofailover.replication_state
+   OUT assigned_group_state pgautofailover.replication_state,
+   OUT candidate_priority	int,
+   OUT replication_quorum	bool
  )
 RETURNS SETOF record LANGUAGE SQL STRICT
 AS $$
-   select nodename, nodeport, groupid, nodeid, reportedstate, goalstate
-     from pgautofailover.node
-    where formationid = formation_id
+   select nodename, nodeport, groupid, nodeid, reportedstate, goalstate,
+   		  candidate_priority, replication_quorum
```

Comment:
Those new output columns are NOT printed in `monitor_print_state` as far as I can see. This is strange, we should either not report them or report them and then print them too. Why report them and not print them? (or did I miss the change in the `monitor_print_state` function maybe?)

Reply:
modified monitor_print_state to display new columns